website/common/locales/en/groups.json

```diff
@@ -132,7 +132,14 @@
   "possessiveParty": "<%= name %>'s Party",
   "clearAll": "Delete All Messages",
   "confirmDeleteAllMessages": "Are you sure you want to delete all messages in your inbox? Other users will still see messages you have sent to them.",
-  "optOutPopover": "Don't like private messages? Click to completely opt out",
+  "PMEnable": "Enable",
+  "PMEnableFull": "Enable private messages",
+  "PMDisable": "Disable",
+  "PMDisableFull": "Disable private messages",
+  "PMDisableConfirmation": "Are you sure you want to disable private messages? You still will be able to send messages, but no one will be able to send them to you.",
+  "PMEnabledOptPopoverText": "When private messages are disabled, you still can send messages, but no one can send them to you.",
+  "PMDisabledOptPopoverText": "Enable this option to be able to receive messages.",
+  "disabledPMCaptionTitle": "Private messages are disabled. You still can send messages, but no one can send them to you.",
```

Comment:
disabledPMCaptionTitle also isn't used at the moment, but it might be useful to display it somewhere on the messages screen so that it's really obvious to the user that no one can reply to them. Maybe underneath the "Send" button and text box? From what I saw on the old site, it wasn't uncommon for people to disable messages and then forget that they'd done so and be confused when people couldn't message them. I'll check with the other mods to see if they also think it will be worth adding the clutter, and then we'll check with Tressley about it. So, don't delete this string just yet. :)